pkg/helm/chartproxy/repos_test.go

```diff
@@ -0,0 +1,57 @@
+package chartproxy
+
+import (
+	"testing"
+
+	"github.com/openshift/console/pkg/helm/actions/fake"
+)
+
+func TestHelmRepoGetter_List(t *testing.T) {
+	tests := []struct {
+		name             string
+		ReposNum         int
+		expectedRepoName []string
+	}{
+		{
+			name:             "return 2 repos found in cluster",
+			ReposNum:         2,
+			expectedRepoName: []string{"sample-repo-1", "sample-repo-2"},
+		},
+		{
+			name:             "return 1 repos found in cluster",
+			ReposNum:         1,
+			expectedRepoName: []string{"sample-repo-1"},
+		},
+		{
+			name:             "return default repo when none are declared in cluster",
+			ReposNum:         0,
```

Comment:
Shouldn't this be  `1` since the default one is returned ?

Reply:
it represents the number of HelmChartRepos found in cluster

Reply:
nope, we are testing the situation when there are no HelmChartRepo CRs available in the cluster - in that case, we should still fall back on the default repo.